templates/flavors/k3s/default/k3sControlPlane.yaml

```diff
@@ -15,29 +15,6 @@ spec:
         content: |
           flannel-backend: none
           disable-network-policy: true
-      - path: /var/lib/rancher/k3s/server/manifests/ccm-linode.yaml
-        owner: root:root
-        content: |-
-          apiVersion: helm.cattle.io/v1
-          kind: HelmChart
-          metadata:
-            namespace: kube-system
-            name: ccm-linode
-          spec:
-            targetNamespace: kube-system
-            version: ${LINODE_CCM_VERSION:=v0.4.4}
-            chart: ccm-linode
-            repo: https://linode.github.io/linode-cloud-controller-manager/
-            bootstrap: true
-            valuesContent: |-
-              routeController:
-                vpcName: ${VPC_NAME:=${CLUSTER_NAME}}
-                clusterCIDR: 10.0.0.0/8
-                configureCloudRoutes: true
-              secretRef:
-                name: "linode-token-region"
-              nodeSelector:
-                node-role.kubernetes.io/control-plane: "true"
```

Comment:
Looks like this node-selector is not getting set correctly when using templates/addons

Reply:
Might need a patch to set the nodeSelector to correct value. Default is set to blank. https://github.com/linode/linode-cloud-controller-manager/blob/main/deploy/chart/values.yaml#L16